core/capabilities/triggers/logevent/service.go

```diff
@@ -0,0 +1,161 @@
+package logevent
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+)
+
+const ID = "log-event-trigger-%s-%d@1.0.0"
+
+const defaultSendChannelBufferSize = 1000
+
+// Log Event Trigger Capability Input
+type Input struct {
+}
+
+// Log Event Trigger Capability Payload
+type Payload struct {
+	// Time that Log Event Trigger's task execution occurred (RFC3339Nano formatted)
+	ActualExecutionTime string
+}
+
+// Log Event Trigger Capabilities Manager
+// Manages different log event triggers using an underlying triggerStore
+type TriggerService struct {
+	capabilities.CapabilityInfo
+	capabilities.Validator[RequestConfig, Input, capabilities.TriggerResponse]
+	lggr           logger.Logger
+	triggers       CapabilitiesStore[logEventTrigger, capabilities.TriggerResponse]
+	relayer        core.Relayer
+	logEventConfig Config
+	stopChan       services.StopChan
+	wg             sync.WaitGroup
+}
+
+// Common capability level config across all workflows
+type Config struct {
+	ChainID        uint64 `json:"chainId"`
+	Network        string `json:"network"`
+	LookbackBlocks uint64 `json:"lookbakBlocks"`
+	PollPeriod     uint32 `json:"pollPeriod"`
+}
+
+func (config Config) Version(capabilityVersion string) string {
+	return fmt.Sprintf(capabilityVersion, config.Network, config.ChainID)
+}
+
+type Params struct {
+	Logger         logger.Logger
+	Relayer        core.Relayer
+	RelayerSet     core.RelayerSet
+	LogEventConfig Config
+}
+
+var _ capabilities.TriggerCapability = (*TriggerService)(nil)
+var _ services.Service = &TriggerService{}
+
+// Creates a new Cron Trigger Service.
+// Scheduling will commence on calling .Start()
+func NewLogEventTriggerService(p Params) *TriggerService {
```

Comment:
We are already in `package logevent` - no need to restate:
```suggestion
func NewTriggerService(p Params) *TriggerService {
```

Reply:
Fixed this